.github/renovate.json5

```
@@ -0,0 +1,37 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended",
+    ":disableRateLimiting",
+    ":noUnscheduledUpdates",
+    // Avoid conflicts with custom `commitMessagePrefix`
+    ":semanticCommitsDisabled"
```

This was an error in my previous commits, this should be this:

```suggestion
    ":semanticCommits"
```